src/main/java/com/google/step/snippet/external/StackOverflowClient.java

```diff
@@ -71,13 +66,12 @@ public Card search(String url) {
       return null;
     }
     String title = getTitle(questionId);
-    String answerBody = getAnswerBody(answerId);
-    if (title == null || answerBody == null) {
+    String description = getAnswerBody(answerId);
+    if (title == null || description == null) {
       return null;
     }
-    // No description or code is allowed for StackOverflow.
-    String description = getDescription(answerBody);
-    String code = getCode(answerBody);
+    // Code is intentionally set to null for better display.
+    String code = null;
```

Comment:
why not just set the param to null in the return statement?

Reply:
Yeah, I think you can just set to null in the return statement. If you do that, can you do add a comment so it's clear which field is null? 

`return new Card(title, /* code= */ null, url, description, SOURCE_NAME, ICON_LINK);`